configs/openmrs_config/roles/roles.csv

```diff
@@ -1,12 +1,12 @@
-640fb1e9-1c27-4a17-b3ad-85eda5a6fe53,Advanced User Admin,Advanced User Admin,,Add Users;View Users;Edit Users;Delete Users;Edit User Passwords,
-8c024914-db55-4b16-8de5-13494c783c0f,Basic User Admin,Basic User Admin,,Add Users;View Users;Edit Users,
-04d20272-297d-44a9-b5e7-a8d980f303ca,Advanced Patient Admin,Advanced Patient Admin,,Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits;Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits,
-416d4331-c217-44b1-a0f9-59d0b6c69475,Basic Patient Admin,Basic Patient Admin,,Add patients;View Patients;Add Visits,
+640fb1e9-1c27-4a17-b3ad-85eda5a6fe53,Advanced User Role,Advanced User Role,,Add Users;View Users;Edit Users;Delete Users;Edit User Passwords,
+8c024914-db55-4b16-8de5-13494c783c0f,Basic User Role,Basic User Role,,Add Users;View Users;Edit Users,
+04d20272-297d-44a9-b5e7-a8d980f303ca,Advanced Patient Role,Advanced Patient Role,,Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits;Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits,
+416d4331-c217-44b1-a0f9-59d0b6c69475,Basic Patient Role,Basic Patient Role,,Add patients;View Patients;Add Visits,
```

Comment:
Should these not changed as earlier suggested i.e,

`Advanced Patient Admin` ➜ `Advanced Provider Role`
`Basic Patient Admin` ➜ `Basic Provider Role`

What this `Advanced Patient Role` intuitively means that a logged-in patient having the role should be able to accomplish somethting?

Reply:
Hi @Ruhanga there were four roles in particular as per the ticket requirement as listed below. I had added those others to just fine tune permissions and they are not necessary so I will discard them altogether and add the privileges they had to the particular role required.

```
Role for CC care providers

Role for OD/PHD users, only for reporting

Role for NCD managers, only for reporting

Role for PMD admin user 
```